package.json

```diff
@@ -65,26 +68,28 @@
     ]
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.15.8",
     "@ember-data/rfc395-data": "^0.0.4",
-    "@glimmer/syntax": "^0.84.2",
+    "@glimmer/syntax": "^0.85.5",
```

Comment:
Hm, this is latest. I guess require can work with an esm module in some situation.

Reply:
mmm, actually didnt test the code...

Reply:
Is there an async location to import this?

Otherwise i wonder if a whole eslint plugin can be esm

Reply:
I think esm eslint plugin can only be used with the new config files.